test/format/failure_configs/CONFIG.WT-6727

```diff
@@ -75,7 +75,6 @@ runs.source=table
 runs.threads=16
 runs.timer=4
 runs.type=row-store
-statistics=0
```

Comment:
I understand opting configs that previously wanted stats into the `all` setting, but do we want to do the same for configs that wanted stats off? Perhaps it's just people copy-pasting, but given that a number of tests opted out of stats, perhaps it's better to change this at the config level. (i.e. leave the `statistics` config item, have 0 continue to mean off, but change 1 to be `all`.)

Reply:
The change itself is to ensure we always to enable stats for test format. I think we should not run with stat config off so I decided to remove it.